README.md

```diff
@@ -1,11 +1,16 @@
-This repository contains the code of the Azure IoT Identity Service and related services. Together these services make a basic device "runtime" for Azure IoT devices.
+# IoT Identity Services for Edge Devices
```

Comment:
It's not just for Edge devices.

Reply:
I'm taking an expansive view here of what is considered an "edge device".